.vscode/settings.json

```diff
@@ -5,5 +5,13 @@
             "--proto_path=${workspaceRoot}/backend",
         ]
     },
-    "go.useLanguageServer": true
+    "go.useLanguageServer": true,
+    "vetur.format.options.tabSize": 4,
+    "prettier.tabWidth": 4,
+    "prettier.packageManager": "yarn",
+    "editor.formatOnSave": true,
+    "javascript.format.enable": false,
+    "vetur.format.defaultFormatter.html": "prettier",
+    "prettier.printWidth": 120,
```

Comment:
Could this be set to 100? I prefer keeping everything under 80 lines, but use it as a soft limit with 100 being a strict limit. It makes everything a lot more readable.

Reply:
No, the tab size is 4 spaces. When you write HTML, you almost immediately reach 80 characters and the HTML becomes unreadable.